bower.json

```diff
@@ -29,13 +29,9 @@
     "paper-listbox": "PolymerElements/paper-listbox#2",
     "paper-tabs": "PolymerElements/paper-tabs#2",
     "paper-toast": "PolymerElements/paper-toast#2",
-    "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
-    "google-youtube": "GoogleWebComponents/google-youtube#^1.2.1",
-    "google-map": "GoogleWebComponents/google-map#^1.1.10",
+    "promise-polyfill": "polymerlabs/promise-polyfill#2",
```

Comment:
I think we can drop the separate promise-polyfill now... There's a promise polyfill in the v1 WC polyfills, so WC-loader should take care of this, I think. (Only needed this to run the hybrid iron-ajax against 1.x, IIRC).